arangod/RestServer/arangod.cpp

```diff
@@ -56,6 +56,7 @@
 #include "Cluster/ClusterFeature.h"
 #include "Cluster/ClusterUpgradeFeature.h"
 #include "Cluster/MaintenanceFeature.h"
+#include "Cluster/FailureOracleFeature.h"
```

Comment:
Why is this included here? It does not seem to be used in the file...

Reply:
It needs to be there for some static assertions in ApplicationServer, fails to compile otherwise.